packages/turbo/src/common.ts

```diff
@@ -49,3 +49,15 @@ export function getNativeModule<T>(moduleName: string): T {
         }
       );
 }
+
+export async function openExternalURL({
+  url,
+}: OpenExternalUrlEvent): Promise<void> {
+  const supported = await Linking.canOpenURL(url);
+
+  if (supported) {
+    await Linking.openURL(url);
```

Comment:
@pklatka Wouldn't we want to return the promise? I guess we don't use it currently but we might use response at some point